docs/zh/latest/stream-proxy.md

```diff
@@ -147,7 +147,7 @@ curl http://127.0.0.1:9080/apisix/admin/stream_routes/1 -H 'X-API-KEY: edd1c9f03
    ERROR 2013 (HY000): Lost connection to MySQL server at 'reading initial communication packet', system error: 2
    ```
 
-   Instead making a request to the APISIX host and port where the filter matching succeeds:
+   而给 APISIX 的主机和端口发出请求，过滤器匹配成功。
```

Comment:
Not too sure 🤔 cc @tzssangglass

Reply:
I see, here seems a little unclear. 

Here we can express: the following request matches stream route 1, so it can be proxied to mysql normally.